### ISSUE TITLE
Training keyframe and interpolation model

### ISSUE THREAD

**Issue by ashwani-ver** [antonibigata/keysync#43]
hi @antonibigata ,
Thanks for making your wonderful work public.
I am trying to train keyframe model and interpolation model on custom hindi data which is around 10 hrs long.
I trained the keyframe model around 10 epochs and then started training the interpolation model. The interpolation model is trained upto 12 epochs. I did inference on the same videos given in training, i find that lip is not syncing with the given audio, though it is able to preserve the identity, but lipsync is not there.
Can you please share me few ideas , how to improve it?

Also, if use pretrained models for inferencing, the model generates good output if lighting condition stable and does not have dark color. But when i try to do dubbing on the videos in which someone is sitting in the dim lighting condition, the output video has dark artifacts over the lip region upto the box masking. Can you please guide me in this regard as well?
Thanks

**Comment by antonibigata** [comment]
Sorry for the late reply.

After writing the paper, I realised that WavLM performed marginally better for the interpolation model, so in the repo I use different audio embeddings for the keyframe model and the interpolation one. Maybe this line needs to be changed: https://github.com/antonibigata/keysync/blob/f7827b041d9d30740a96998f0f9946ae19b2d248/scripts/sampling/dubbing_pipeline.py#L500-L503
